src/datasources/exchange-api/exchange-api.service.spec.ts

```diff
@@ -51,20 +56,6 @@ describe('ExchangeApi', () => {
     expect(fiatCodes).toBe(expectedFiatCodes);
   });
 
-  it('fiatCodes uses default cache TTL (12h) if none is set', async () => {
-    const expectedFiatCodes = exchangeFiatCodesBuilder().build();
-    mockCacheFirstDataSource.get.mockResolvedValue(expectedFiatCodes);
-
-    await service.getFiatCodes();
-
-    expect(mockCacheFirstDataSource.get).toBeCalledWith(
-      new CacheDir('exchange_fiat_codes', ''),
-      `${exchangeBaseUri}/symbols`,
-      { headers: { apikey: exchangeApiKey } },
-      60 * 60 * 12, // 12h in seconds
-    );
-  });
```

Comment:
This test is no longer needed as it was testing the default set in the class. However, the default is not the class' responsibility anymore.